docs/apis/cli.md

```diff
@@ -185,6 +189,18 @@ Action "list" lists running and scheduled programs.
      -s,--scheduled                Show only scheduled programs and their JobIDs
 
 
+Action "stop" stops a running program (streaming jobs only).
+
+  Syntax: stop [OPTIONS] <Job ID>
+  "stop" action options:
+     -m,--jobmanager <host:port>   Address of the JobManager (master) to which
+                                   to connect. Specify 'yarn-cluster' as the
+                                   JobManager to deploy a YARN cluster for the
+                                   job. Use this flag to connect to a different
```

Comment:
It would be nice if @mjsax could remove the sentence: 
"Specify 'yarn-cluster' as the JobManager to deploy a YARN cluster for the job. "
from the description. The "stop" command is not about deploying a job ;)